src/core/xref.js

```diff
@@ -72,6 +84,72 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+function invalidProfileError(profile) {
+  pub(
+    "error",
+    `Invalid cross-references profile "${profile}". Please use one of the supported specification profiles: ${Object.keys(
```

Comment:
Note, users generally won't know that "cross-references" means. So always link it back to things they can find and fix.

```suggestion
    `Invalid `xref` profile option passed in the configuration: "${profile}". Please use one of the supported specification profiles: ${Object.keys(
```